.github/workflows/test-on-push.yml

```diff
@@ -0,0 +1,165 @@
+name: Super ultra fast test, lint and visual regression
+
+on: push
```

Comment:
Do you have a suggestion on how this should work?

Reply:
Not sure, would need to investigate. It might be a combination of push and pull_request. Note how the "push" job has triggered actions alex-page/polaris-react but there's been no action triggered within the context of this repo. I think that means we wouldn't be able to enforce CI passing on PRs from forks atm as nothing gets mentioned in the list of checks at the bottom of this PR.

Reply:
Yeah I tested this and you are right. I created a fork of [alex-page/harmonograph-xy/](https://github.com/alex-page/harmonograph-xy/) and the pull request did not trigger the `on: push`.